boskos/boskos_test.go

```diff
@@ -495,28 +495,28 @@ func TestUpdate(t *testing.T) {
 			method:    http.MethodPost,
 		},
 		{
-			name:      "no name",
+			name:      "missing name",
 			resources: []common.Resource{},
 			path:      "?state=s&owner=merlin",
 			code:      http.StatusBadRequest,
 			method:    http.MethodPost,
 		},
 		{
-			name:      "no owner",
+			name:      "missing owner",
```

Comment:
a) What is missing an owner? The request or the resources or?
b) Consider state what should happen when the owner is missing (aka reject the request)